Wikipedia/UI-V5/WMFNearbyViewModel.m

```diff
@@ -0,0 +1,193 @@
+//
+//  WMFNearbyViewModel.m
+//  Wikipedia
+//
+//  Created by Brian Gerstle on 9/9/15.
+//  Copyright (c) 2015 Wikimedia Foundation. All rights reserved.
+//
+
+#import "WMFNearbyViewModel.h"
+
+// Location & Fetching
+#import "WMFLocationManager.h"
+#import "WMFLocationSearchFetcher.h"
+
+// Models
+#import "WMFLocationSearchResults.h"
+#import "MWKLocationSearchResult.h"
+#import "MWKHistoryEntry.h"
+#import "CLLocation+WMFApproximateEquality.h"
+#import "CLLocation+WMFBearing.h"
+
+// Frameworks
+#import "Wikipedia-Swift.h"
+#import "PromiseKit.h"
+
+@interface WMFNearbyViewModel ()
+<WMFNearbyControllerDelegate>
+
+@property (nonatomic, strong) WMFLocationSearchFetcher* locationSearchFetcher;
+@property (nonatomic, strong) WMFLocationManager* locationManager;
+@property (nonatomic, assign) NSUInteger resultLimit;
+
+@property (nonatomic, strong, readwrite) WMFLocationSearchResults* locationSearchResults;
+
+@property (nonatomic, strong) NSMutableIndexSet* autoUpdatingIndexes;
```

Comment:
See the header for methods that start/stop auto updating.  I should've added comments, but it's basically a set of indexes which we're currently updating in response to location & heading changes. The API in the header is documented and provide some rationale.